libs/core/langchain_core/chat_history.py

```diff
@@ -138,6 +155,23 @@ def add_ai_message(self, message: Union[AIMessage, str]) -> None:
         else:
             self.add_message(AIMessage(content=message))
 
+    async def aadd_ai_message(self, message: Union[AIMessage, str]) -> None:
```

Comment:
add_ai_message should be treated as deprecated -- we'll likely deprecate it sometime soon, so let's not introduce `add_ai_message` here

Reply:
done